### PR TITLE
chore: update `engines` versions in `package.json`

### PR DESCRIPTION
**Pull request by QuiiBz**
## Type

- Chore

#### What is expected?

Update `engines` versions in `package.json` to match our other repositories.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
   ],
   "packageManager": "pnpm@7.20.0",
   "engines": {
-    "node": ">=18.x",
-    "pnpm": ">=7.0.0"
+    "node": ">=v18.12.1",
+    "pnpm": ">=7.20.0"
```
Comment (chambo-e) [inline package.json:11]: This is a rather very strict policy, are we sure we need this ?

Comment (QuiiBz) [reply package.json:11]: This is what we have in all our other projects. We also already set the `packageManager` field to `pnpm@7.20.0`, so if we want to allow broader versions we should also update it.

File: package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
   ],
   "packageManager": "pnpm@7.20.0",
   "engines": {
-    "node": ">=18.x",
-    "pnpm": ">=7.0.0"
+    "node": ">=v18.12.1",
```
Comment (matthprost) [inline package.json:10]: Why should we force for this specific version instead of `v18.x` ?

**Review by matthprost — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (QuiiBz) [reply package.json:10]: See https://github.com/scaleway/scaleway-ui/pull/2161#discussion_r1060101653

**Comment by QuiiBz** [comment]
@chambo-e @matthprost I've updated the versions to `major.x` as discussed.

File: package.json (hunk 2 above)
Comment (QuiiBz) [reply package.json]: Update: https://github.com/scaleway/scaleway-ui/pull/2161#issuecomment-1369775010

File: package.json (hunk 1 above)
Comment (QuiiBz) [reply package.json]: Update: https://github.com/scaleway/scaleway-ui/pull/2161#issuecomment-1369775010

**Review by chambo-e — APPROVED** [review]

**Review by matthprost — APPROVED** [review]

**Pull request merged**